app/Services/ForwardChatworkService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Webhook;
+use App\Enums\PayloadHistoryStatus;
+
+class ForwardChatworkService
+{
+    protected $webhook;
+    protected $params;
+    protected $payloadHistoryRepository;
+
+    public function __construct(Webhook $webhook, $params, $payloadHistoryRepository)
+    {
+        $this->webhook = $webhook;
+        $this->params = $params;
+        $this->payloadHistoryRepository = $payloadHistoryRepository;
+    }
+
+    public function call()
+    {
+        $payloads = $this->webhook->payloads;
+        $params = $this->params;
+        $filledPayloads = [];
+        foreach ($payloads as $payload) {
+            $conditions = $payload->conditions;
+            $isValid = true;
+            foreach ($conditions as $condition) {
+                // 'return $payloads->fieldName == "value";'
+                $expression = 'return ' . $condition->field . ' '
+                                        . $condition->operator . ' "'
+                                        . $condition->value . '";';
+                if (!eval($expression)) {
+                    $isValid = false;
+                    break;
+                }
+            }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Services/ForwardChatworkService.php</br>**Line**: 40</br>Whitespace found at end of line